### ISSUE TITLE
Swiper with navigation bar?

### ISSUE THREAD

**Issue by sunweiyang** [leecade/react-native-swiper#659]
I need to create a 4-page swiper with a navigation bar at the bottom, like the one shown here. (The user needs to be able to use either swiping or tapping a navigation bar button to get to the desired page. Is there a way to do this using react-native-swiper?

![image](https://user-images.githubusercontent.com/12490625/32741631-bbf1c9bc-c85b-11e7-8bfc-9fa8df1c9fee.png)

**Comment by seankwalker** [comment]
Yes, you can make the tab bar as a separate component, and each icon defines some page (0, 1, 2, 3, for instance). Then, when you tap an icon, you can calculate the offset (e.g. scrolling from Home (0) to History (2) would be 2) and use the `scrollBy` method with that offset.

**Comment by chandonn** [comment]
Maybe you could use the [react-native-router-flux](https://github.com/aksonov/react-native-router-flux) to set up a navigation bar. After that, using the scenes approach, each set of slides would be one scene:

Something like this:
`<Router navBar={NavBar}>
       <Scene key="Home" component={Home} hideNavBar={true} initial={true}/>
       <Scene key="Skill" component={Skill} hideNavBar={true}/>
       <Scene key="History" component={History} hideNavBar={true}/>
       <Scene key="Profile" component={Profile} hideNavBar={true}/>
</Router>`

Each `component` should be one separate set of slides, and `NavBar` would have the styles and icons

**Comment by machester4** [comment]
to implement scrollBy here I leave a function that will help you 

`
**// index = position of the page
//  this.sw = local reference from swiper component**

scrollTo = index => {
    let currentScroll = this.state.indexSlide; // you can get using onIndexChanged
    let animation = true; 
    if(index == 0){
      this.sw.scrollBy(-(currentScroll + index), animation);
    }else if(index == (currentScroll + 1)){
      console.log('adelante');
      this.sw.scrollBy(1, animation);
    } else if(index == (currentScroll - 1)){
      console.log('atras');
      this.sw.scrollBy(-1, animation);
    } else if(index > currentScroll) {
      this.sw.scrollBy(index, animation);
    } else if(index < currentScroll){
      this.sw.scrollBy(-(index + 1), animation);
    }
  };`
